docs/privacy_policy.md

```diff
@@ -0,0 +1,54 @@
+## Privacy Policy
+
+Georepublic UG (haftungsbeschränkt) built the Consento app as an Open Source app. This SERVICE is provided by Georepublic UG (haftungsbeschränkt) at no cost and is intended for use as is.
+
+This page is used to inform visitors regarding our policies with the collection, use, and disclosure of Personal Information if anyone decided to use our Service.
+
+If you choose to use our Service, then you agree to the collection and use of information in relation to this policy. The Personal Information that we collect is used for providing and improving the Service. We will not use or share your information with anyone except as described in this Privacy Policy.
+
+The terms used in this Privacy Policy have the same meanings as in our Terms and Conditions, which is accessible at Consento unless otherwise defined in this Privacy Policy.
+
+**Information Collection and Use**
+
+For a better experience, while using our Service, we may require you to provide us with certain personally identifiable information. The information that we request will be retained by us and used as described in this privacy policy.
+
+The app does use third party services that may collect information used to identify you.
+
+Link to privacy policy of third party service providers used by the app
+
+**Log Data**
+
+We want to inform you that whenever you use our Service, in a case of an error in the app we collect data and information (through third party products) on your phone called Log Data. This Log Data may include information such as your device Internet Protocol (“IP”) address, device name, operating system version, the configuration of the app when utilizing our Service, the time and date of your use of the Service, and other statistics.
+
+**Service Providers**
+
+We may employ third-party companies and individuals due to the following reasons:
+
+*   To facilitate our Service;
+*   To provide the Service on our behalf;
+*   To perform Service-related services; or
+*   To assist us in analyzing how our Service is used.
```

Comment:
We are working as hard as we can to not allow tracking through the app, I think this is not our goal.

Reply:
..."in a case of an error in the app we collect data and information (through third party products) on your phone called Log Data."

I think this can be changed a bit. 

"...in the case of an error in the app, we collect data and information with third party products on your phone. This is called Log Data."

or

"... in the occurrence of an error in the app, we collect data and information (through third party products) on your phone. This is called Log Data."

It is not too important, but separating this long sentence into two sentences makes it a little easier to understand. Also, the comma is important here.